lib/runtime/wasmer/instance.go

```diff
@@ -43,7 +43,7 @@ type Instance struct {
 	ctx      *runtime.Context
 	isClosed bool
 	codeHash common.Hash
-	sync.Mutex
+	mutex    sync.Mutex
```

Comment:
Note this was changed since I was having trouble finding references to the Lock/Unlock methods.

As a side note, I think we should avoid embedding things due to that, and also because it exports methods we might not want exported actually (like here, the mutex should be private).